solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -208,6 +214,12 @@ contract RandomBeacon is Ownable {
         address indexed challenger
     );
 
+    event DkgResultMaliciousSlashed(
```

Comment:
I'd rather say `DkgMaliciousResultSlashed` because it's the result that was malicious, not the slashing 😉

Reply:
See https://github.com/keep-network/keep-core/pull/2695/commits/f0db676b0854612c89aabe9532493efb07dbd660